hack/init-environment.sh

```diff
@@ -178,6 +180,40 @@ function install_flux () {
     --set='rbac.pspEnabled=true'
 }
 
+#####
+## Set up and install the Vault secret manager in the command center GKE cluster.
+#####
+function install_secret_manager () {
+  local -r kubeconfig=$1 env_dir=$2 env=$3
+
+  # Install the CRD definitions separately. Helm doesn't have
+  # a coherent story for handling these yet (since updating them
+  # the wrong way can result in all existing objects being deleted),
+  # so they're easier to handle out-of-band.
+  declare -ra kubernetes=($(configure_kubernetes ${kubeconfig}))
+  ${kubernetes[@]} apply -f \
+    https://raw.githubusercontent.com/tuenti/secrets-manager/${SECRET_MANAGER_VERSION}/config/crd/bases/secrets-manager.tuenti.io_secretdefinitions.yaml
+  # Install the Operator using Helm.
+  declare -ra helm=($(configure_helm ${kubeconfig} ${env_dir}))
+
+  rm -rf ${env_dir}/.helm
+  # helm repo adds Jade’s Helm repository
+  ${helm[@]} repo add add datarepo-helm https://broadinstitute.github.io/datarepo-helm
+  # helm repo upgrade --installs the secret manager chart, setting appropriate values
+  ${helm[@]} upgrade install-secrets-manager datarepo-helm/install-secrets-manager \
+    --install \
+    --wait \
+    --namespace=secrets-manager \
+    --version=${SECRET_MANAGER_CHART_VERSION} \
+    --set='installcrd.install=false' \
+    --set='vaultLocation=https://clotho.broadinstitute.org:8200' \
+    --set='vaultVersion=kv2' \
+    --set='serviceAccount.create=true' \
+    --set='rbac.create=true' \
+    --set='secretsgeneric.roleId=$(vault read -field=role_id secret/dsde/monster/${env}/approle-monster-${env})' \
+    --set='secretsgeneric.secretId=$(vault read -field=secret_id secret/dsde/monster/${env}/approle-monster-${env})'
```

Comment:
I think the single-quotes on these two `--set`s need to be double-quotes for the variable expansion to work.

Reply:
correct anything in singles will be taken literally